src/internal/hooks/use-performance-marks.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
For now its probably fine to have this here. I imagine it'll move to component-toolkit eventually once we want to share with the board components package.

Reply:
I agree, we'll probably have to move it in the future.